services/src/main/java/org/keycloak/protocol/oidc/endpoints/request/AuthzEndpointRequestParser.java

```diff
@@ -46,8 +46,11 @@
      */
     public static final int ADDITIONAL_REQ_PARAMS_MAX_SIZE = 200;
 
+    public static final String AUTHZ_REQUEST_PARAMS = "RawRequesParameters";
```

Comment:
I am not sure about introducing constant AUTHZ_REQUEST_PARAMS and the attribute in KeycloakSession for this. It seems to me that requestParameters can be quite useful thing in general for ClientExecutors, so maybe the requestPArameters can be added directly as a field to the AuthorizationRequestContext class?

On the other hand, the "request" object itself (AUTHZ_REQUEST_OBJECT) is not be so widely used IMO, so I agree to keep it as an attribute in KeycloakSession as you did in this PR.

Reply:
@mposolda Yes, I agree with you. I'll add requestParameters to AuthorizationRequestContext while request object is kept on as an attribute.